### ISSUE TITLE
No mention of Nullarbor?

### ISSUE THREAD

**Issue by tseemann** [nmquijada/tormes#4]
TORMES is very similar to Nullarbor https://github.com/tseemann/nullarbor and has existed since Jan 2015 but has not been published.

There is no mention of Nullarbor in your paper or documentation.  

Can you explain why?

**Comment by nmquijada** [comment]
Dear Torsten Seemann,

We have being trying to contact you regarding the TORMES project and manuscript since July 2017, willing for your collaboration. However, we did not got any answer.
We apologize regarding Nullarbor citation. It is a big, although non-intended, mistake. We have already started actions in order to modify the manuscript and the documentation accordingly.

**Comment by tseemann** [comment]
@nmquijada has there been any progress on this matter?

**Comment by andrewjpage** [comment]
I'd be interested to know as well.

**Comment by nmquijada** [comment]
Yes. Modifications have been applied to the documentation and a modified version of the manuscript has been forwarded to the journal's editors. The manuscript is still in the online version and changes will be applied soon. We are keeping track of the process so we'll ensure that all changes have been applied.
As we said, it was a non-intended mistake and we apologize for the inconveniences. 
However, modifications have been issued on time and we hope that all changes will take effect soon.

**Comment by tseemann** [comment]
Thank you  @nmquijada

**Comment by nmquijada** [comment]
Dear tseemann,
We already received the answer from the editors of the journal after several discussions and sadly they issued that although the corrected proof of the manuscript is still in its online version, no further actions can be taken. 
We apologize for this issue. We always try to be very respectful with citing other works. We have already included nullarbor citation in the current tormes documentation and so will be in further documents we are preparing.
During the software and manuscript preparation, we contacted you several times by email. Some of the authors met you some years ago and we wanted to collaborate with you since then and of course not to cause any inconveniences. In some of those emails we shared the manuscript that was going to be submitted with you. However, we did never get any answer. 
Were you aware of these emails?
